src/libcore/iter/iterator.rs

```diff
@@ -2463,6 +2463,52 @@ pub trait Iterator {
             }
         }
     }
+
+    /// Extends passed collection with the contents of an iterator.
```

Comment:
Missing `the` between `passed` and `collection` here and on `extend_mut` as well.